scarb/tests/lint.rs

```diff
@@ -580,3 +580,136 @@ fn lint_selected_targets() {
 
       "#});
 }
+
+#[test]
+fn lint_specific_file() {
+    let t = TempDir::new().unwrap();
+    ProjectBuilder::start()
+        .name("hello")
+        .lib_cairo(indoc! {r#"
+            mod other;
+
+            fn main() {
+                let x = true;
+                if x == false {
+                    println!("x is false");
+                }
+            }
+        "#})
+        .build(&t);
+
+    t.child("src/other.cairo")
+        .write_str(indoc! {r#"
+        fn main() {
+            loop {
+                break ();
+            }
+        }
+    "#})
+        .unwrap();
+
+    Scarb::quick_snapbox()
+        .arg("lint")
+        .arg("src/other.cairo")
+        .current_dir(&t)
+        .assert()
+        .success()
+        .stdout_matches(indoc! {r#"
+             Linting hello v1.0.0 ([..]/Scarb.toml)
+        warn: Plugin diagnostic: unnecessary double parentheses found after break. Consider removing them.
+         --> [..]/src/other.cairo:3:9
+                break ();
+                ^^^^^^^^^
+
+        "#});
+}
+
+#[test]
+fn lint_specific_module() {
```

Comment:
Nitpick:
```suggestion
fn lint_specific_directory() {
```
(as it's not semantic)